pprofile.py

```diff
@@ -883,14 +962,37 @@ def runpath(path, argv, filename=None, threads=True, verbose=False):
     """
     _run(threads, verbose, 'runpath', filename, path, argv)
 
-_allsep = os.sep + (os.altsep or '')
+def _qCacheGringAdapt(path):
+    '''
+    qCacheGrind (windows build) needs at least one UNIX separator
+    in path to find the file. Adapt here even if this is probably
+    more of a qCacheGrind issue...
+    '''
+
+    if os.path.sep != "/":
```

Comment:
So this code is calculated only when `_path_resolver()` is called, and `_path_resolver()` is only called from 3 places (see my response to your next comment) - and these are not inside loops - i.e., they're only called once per user execution of `pprofile`.

Please correct me if I'm wrong though.